README.md

```diff
@@ -28,7 +28,7 @@ This is a new project based around Pa11y, code-named "Sidekick". Pa11y Sidekick
 
 ## Requirements
 
-This application requires [Node.js] 4+ and [PostgreSQL] 9.4+.
+This application requires [Node.js] 4+ and [PostgreSQL] 9.5+.
```

Comment:
We dropped Node 4 on pa11y a while ago, it's now Node 8+
```suggestion
This application requires [Node.js] 8+ and [PostgreSQL] 9.5+.
```

Just checking the travis.yml as well, and that's Node 8 anyway!! 🎉